src/utilities/theme/utils.ts

```diff
@@ -26,18 +26,28 @@ export function buildCustomProperties(
   newDesignLanguage: boolean,
   tokens?: Record<string, string>,
 ): CustomPropertiesLike {
-  const {colors = {}, colorScheme, config, frameOffset = 0} = themeConfig;
+  const {
+    colors = {},
+    colorScheme,
+    config,
+    frameOffset = 0,
+    navWidth = 240,
```

Comment:
Default `240` plaris nav value [`$navigation-width`](https://github.com/Shopify/polaris-react/pull/3273/files#diff-9085450f12ece7394e7c8d60ccced256R1) used to hold.